.github/workflows/ci-testing.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           pip list
 
       - name: Tests
-        timeout-minutes: 10
+        timeout-minutes: 20
```

Comment:
I believe that 20 min is still very reasonable time as it is not average but a limit time